vendor/engines/secure_rooms/app/views/secure_rooms/shared/_tabnav_secure_room.html.haml

```diff
@@ -0,0 +1,2 @@
+- if @product.is_a?(SecureRoom)
```

Comment:
I'm unsure of this, since the secure room name is baked into the route below and the file name, so it feels a little like we're asking the wrong question